src/frontend/Komponenter/Behandling/VedtakOgBeregning/Barnetilsyn/KontantstøtteValg.tsx

```diff
@@ -195,11 +195,12 @@ const KontantstøtteValg: React.FC<Props> = ({
                         );
                     })}
                     <ContainerMedLuftUnder>
-                        <LeggTilKnapp
-                            onClick={() => kontantstøttePerioder.push(tomKontantstøtteRad)}
-                            knappetekst="Legg til periode"
-                            hidden={!behandlingErRedigerbar}
-                        />
+                        {behandlingErRedigerbar && (
+                            <LeggTilKnapp
```

Comment:
Det er sikkert bedre slik, men vi har `hiddenIf` på f.eks `LeggTilKnapp` og mange andre komponenter som trygt kan brukes. Det er våre komponenter som implementerer dette, ikke designsystemet :)

Reply:
Etter diskusjon med Charlie bestemmer vi oss for å skrinnlegge hele `hiddenIf` komponenten da den innfører unødvendig kompleksitet i koden